sections/semantics-tabular-data.include

```diff
@@ -2442,143 +2480,150 @@ the cell that corresponds to the values of the two dice.
   The following shows how might one mark up the bottom part of table 45 of the <cite>Smithsonian
   physical tables, Volume 71</cite>:
 
-  <pre highlight="html">
-&lt;table&gt;
-  &lt;caption&gt;Specification values: &lt;b&gt;Steel&lt;/b&gt;, &lt;b&gt;Castings&lt;/b&gt;,
-  Ann. A.S.T.M. A27-16, Class B;* P max. 0.06; S max. 0.05.&lt;/caption&gt;
-  &lt;thead&gt;
-  &lt;tr&gt;
-    &lt;th rowspan=2&gt;Grade.&lt;/th&gt;
-    &lt;th rowspan=2&gt;Yield Point.&lt;/th&gt;
-    &lt;th colspan=2&gt;Ultimate tensile strength&lt;/th&gt;
-    &lt;th rowspan=2&gt;Per cent elong. 50.8mm or 2 in.&lt;/th&gt;
-    &lt;th rowspan=2&gt;Per cent reduct. area.&lt;/th&gt;
-  &lt;/tr&gt;
-  &lt;tr&gt;
-    &lt;th&gt;kg/mm&lt;sup&gt;2&lt;/sup&gt;&lt;/th&gt;
-    &lt;th&gt;lb/in&lt;sup&gt;2&lt;/sup&gt;&lt;/th&gt;
-  &lt;/tr&gt;
-  &lt;/thead&gt;
-  &lt;tbody&gt;
-  &lt;tr&gt;
-    &lt;td&gt;Hard&lt;/td&gt;
-    &lt;td&gt;0.45 ultimate&lt;/td&gt;
-    &lt;td&gt;56.2&lt;/td&gt;
-    &lt;td&gt;80,000&lt;/td&gt;
-    &lt;td&gt;15&lt;/td&gt;
-    &lt;td&gt;20&lt;/td&gt;
-  &lt;/tr&gt;
-  &lt;tr&gt;
-    &lt;td&gt;Medium&lt;/td&gt;
-    &lt;td&gt;0.45 ultimate&lt;/td&gt;
-    &lt;td&gt;49.2&lt;/td&gt;
-    &lt;td&gt;70,000&lt;/td&gt;
-    &lt;td&gt;18&lt;/td&gt;
-    &lt;td&gt;25&lt;/td&gt;
-  &lt;/tr&gt;
-  &lt;tr&gt;
-    &lt;td&gt;Soft&lt;/td&gt;
-    &lt;td&gt;0.45 ultimate&lt;/td&gt;
-    &lt;td&gt;42.2&lt;/td&gt;
-    &lt;td&gt;60,000&lt;/td&gt;
-    &lt;td&gt;22&lt;/td&gt;
-    &lt;td&gt;30&lt;/td&gt;
-  &lt;/tr&gt;
-  &lt;/tbody&gt;
-&lt;/table&gt;
-    </pre>
+  <xmp highlight="html">
+    <table>
+      <caption>Specification values: <b>Steel</b>, <b>Castings</b>, Ann. A.S.T.M. A27-16, Class B;* P max. 0.06; S max. 0.05.</caption>
+      <thead>
+        <tr>
+          <th rowspan=2>Grade.</th>
```

Comment:
not sure, but wondering whether 2 should be in quotes or is it as is intentionally? (also next example)

Reply:
I missed it. My grep was looking for words after =. Didn't think about numbers! D'oh!